pandas/tests/test_window.py

```diff
@@ -714,6 +714,16 @@ def test_iter_raises(self, klass):
         with pytest.raises(NotImplementedError):
             iter(obj.expanding(2))
 
+    def test_expanding_axis(self):
```

Comment:
can you parameterize on axis, also can you test this for rolling

Reply:
Yep, done.